packs/data/age-of-ashes-bestiary.db/rinnarv-bontimar.json

```diff
@@ -8829,7 +8829,7 @@
                     "value": {}
                 },
                 "description": {
-                    "value": "<p>You ward a creature against a specified alignment. Choose chaotic, evil, good, or lawful when you cast this spell. The target gains a +1 status bonus to Armor Class and saving throws against creatures and effects of the chosen alignment. This bonus increases to +3 against effects from such creatures that would directly control the target and against attacks made by summoned creatures of the chosen alignment.</p>\n<p>This spell gains the trait that opposes the alignment you chose-if you choose chaos, this spell gains the lawful trait, and vice versa; if you choose evil, this spell gains the good trait, and vice versa.</p>\n@Compendium[pf2e.spell-effects.Spell Effect: Protection]{Spell Effect: Protection}</p>"
+                    "value": "<p>You ward a creature against a specified alignment. Choose chaotic, evil, good, or lawful when you cast this spell. The target gains a +1 status bonus to Armor Class and saving throws against creatures and effects of the chosen alignment. This bonus increases to +3 against effects from such creatures that would directly control the target and against attacks made by summoned creatures of the chosen alignment.</p>\n<p>This spell gains the trait that opposes the alignment you chose-if you choose chaos, this spell gains the lawful trait, and vice versa; if you choose evil, this spell gains the good trait, and vice versa.</p>\n@Compendium[pf2e.spell-effects.Spell Effect: Protection]{Spell Effect: Protection}"
```

Comment:
The ending /p tag shouldn't be removed, rather a p tag should be added before the link.

Reply:
This is an auto-correction, but happy to accept your manual tweaks.